nyaa/views/main.py

```diff
@@ -204,6 +212,10 @@ def home(rss):
 
 
 def render_rss(label, query, use_elastic, magnet_links=False):
+    # Strip invalid XML characters from title.
+    for item in query:
+        item.display_name = escape_xml_illegal_chars(item.display_name)
```

Comment:
This is scary, you sure you want to do this instead of escaping the title in the template?

Reply:
i mean it works fine here

Reply:
Yes, but you're mutating the database models (the ES wrapper objects are whatever), which is scary because they're in the session and all. I'd rather just use `escape_xml_illegal_chars` in the RSS template, so nothing is being mutated.

Reply:
+1 with AMM here, this definitely belongs in the template.